incubator/druid/templates/configmap.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "druid.name" . }}
+  labels:
+    app: {{ template "druid.name" . }}
+    chart: {{ template "druid.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+{{ toYaml .Values.configVars | indent 2 }}
+{{- if .Values.zookeeper.enabled }}
+  druid_zk_service_host: {{ .Release.Name }}-zookeeper-headless:2181
+{{- else }}
+  druid_zk_service_host: {{ .Values.zkHosts }}
+{{- end }}
+{{- if .Values.mysql.enabled }}
```

Comment:
How to use this Chart with external mysql or postgresql ?

Reply:
all druid configuration variables can be set from the `configVars` section of values.yaml. 

For instance, for an external postgres instance I would set: 

```
configVars:
  druid_metadata_storage_type: postgresql
  druid_metadata_storage_connector_connectURI: jdbc:postgresql://postgres:5432/druid
  druid_metadata_storage_connector_user: druid
  druid_metadata_storage_connector_password: FoolishPassword
```

Any suggestions to make this more explicit?